apps/backend/src/mcp/resources/proxy_configs.py

```
@@ -0,0 +1,634 @@
+                }
+                
+                # Calculate distributions
+                for config in db_configs:
+                    status = config.status
+                    sync_status = config.sync_status
+                    
+                    database_info['status_distribution'][status] = database_info['status_distribution'].get(status, 0) + 1
+                    database_info['sync_status_distribution'][sync_status] = database_info['sync_status_distribution'].get(sync_status, 0) + 1
+                    
+                    if config.sync_last_checked:
+                        if not database_info['last_sync'] or config.sync_last_checked > database_info['last_sync']:
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)